assets/src/blocks/ENForm/ENFormInPlaceEdit.js

```diff
@@ -156,7 +156,6 @@ const SideContent = ({attributes, setAttributes}) => {
           placeholder={__('Enter title', 'planet4-blocks-backend')}
           withoutInteractiveFormatting
           allowedFormats={[]}
-          multiline="false"
```

Comment:
In this file we still have several instances of `multiline` (but set to true instead) that should be removed too